components/Textarea/Textarea.tsx

```diff
@@ -17,7 +17,6 @@ const StyledTextarea = styled('textarea', {
   fontFamily: '$rubik',
   margin: '0',
   outline: 'none',
-  padding: '0',
```

Comment:
:information_source: removing this reset as we already have `p: '$3'` [here](https://github.com/traefik/faency/pull/295/files#diff-f5d77bd4b99adef81794a7cae30e4c6d288586702603dd0f89daa714b0e2598fR24)